### ISSUE TITLE
"please check your gas amount" when migrating to quorum

### ISSUE THREAD

**Issue by moda20** [trufflesuite/trufflesuite.github.io#222]
hello !
I am trying to migrate a contract to the 7 nodes example from the quorum repository. 
i  got this error 
```

Error encountered, bailing. Network state unknown. Review successful transactions manually.
Error: The contract code couldn't be stored, please check your gas amount.
```
How is this gas related error even possible, isn't quorum letting the gas thing and the mining for some other ways ? 
is this a generic error

**Comment by cgewecke** [comment]
@moda20 Yes, this is a generic error thrown by `web3` when it tries to new a contract but can't retrieve the contract code from the network after the transaction completes.  This can be thrown for non-gas related reasons - a common one is that the contract's constructor contains `require` statements that throw an error. 

Not sure what client you are using but I believe it's necessary to run Parity with a Geth mode flag to reliably avoid this error as well.

**Comment by moda20** [comment]
@cgewecke i am running the 7 nodes example form the official quorum repo, and that launches geth nodes. I am just using truffle from terminal and never used parity, but i will check my code more. at least now i know it is not gas problem.
